assets/javascripts/initializers/communitarian.js.es6

```diff
@@ -90,6 +93,24 @@ function initializeCommunitarian(api) {
   });
 
   api.modifyClass("controller:create-account", {
```

Comment:
Maybe move this modifyClass invocation to another initializer to reduce file size?